gloo/docs/user_guides/node_port.md

```diff
@@ -1,37 +1,43 @@
 ---
-title: Exposing Gloo with NodePort
+title: Sandbox Deployment Using NodePort
 weight: 5
 ---
 
 ## Motivation
-By default, micro-services deployed in kubernetes have an internal flat network that is not accessible from the outside
-of the cluster. This is true even if you use kuberentes on a public cloud (like Amazon AWs or Google Cloud).
+By default, microservices deployed in Kubernetes have an internal flat network that is not accessible from the outside
+of the cluster. This is true even if you use Kubernetes on a public cloud (like Amazon AWS or Google Cloud).
 
-A NodePort is a way to make kubernetes services available from outside the cluster (and potentially allow access from the internet).
+A NodePort service is a way to make Kubernetes services available from outside the cluster (and potentially allow access from the internet) by opening ports on all of the nodes in the cluster and allowing traffic to go directly to the pods running within the cluster.
 
-In this document we will review how to expose Gloo via a NodePort.
+In this document, we will review how to expose Gloo via a NodePort service. It's important to note that NodePort is *not* a recommended production setting. Node port has the following drawbacks not limited to:
+
+* You must coordinate what services use what ports so there is no conflict
```

Comment:
not sure one needs to manually coordinate as usually kubernetes does it for you

Reply:
If you really try to use nodeport in production, those ports are tightly controlled by firewalls and other operational mechanisms. They are opened one at a time and under tight scrutiny. They would need to be managed and explicitly exposed, not left to k8s to do. In non-prod usage,

Reply:
makes sense